src/core/annotations/qgsannotationmarkeritem.h

```diff
@@ -41,21 +41,21 @@ class CORE_EXPORT QgsAnnotationMarkerItem : public QgsAnnotationItem
 
     QString type() const override;
     void render( QgsRenderContext &context, QgsFeedback *feedback ) override;
-    bool writeXml( QDomElement &element, QDomDocument &document, const QgsReadWriteContext &context ) const override;
-    Qgis::AnnotationItemFlags flags() const override;
-    QList< QgsAnnotationItemNode > nodes() const override;
-    Qgis::AnnotationItemEditOperationResult applyEdit( QgsAbstractAnnotationItemEditOperation *operation ) override;
-    QgsAnnotationItemEditOperationTransientResults *transientEditResults( QgsAbstractAnnotationItemEditOperation *operation ) override SIP_FACTORY;
+    virtual bool writeXml( QDomElement &element, QDomDocument &document, const QgsReadWriteContext &context ) const override;
```

Comment:
why adding virtual here ? override already indicates this (not sure about QGIS habits though)